### ISSUE TITLE
Responsiveness of multiple fields: field has an unwanted left margin

### ISSUE THREAD

**Issue by edum18** [buefy/buefy#1228]
### Overview of the problem

**Buefy** version: 0.7.2
**Vuejs** version: 2.6.6
**OS/Browser**: Windows10/Chrome

### Steps to reproduce

codesandbox:
https://codesandbox.io/s/3ynz8zvo3q

### Expected behavior

By shrinking page, the second input shouldn't have the left margin.

### Actual behavior

By shrinking page, the second input has a left margin.

Should I just use columns then? Doesn't seem right since there is `grouped group-multiline` options.

**Comment by yxngl** [comment]
Targeting wrapped elements seems hard. May be fixed by changing left margin of 0.75rem to right margin for nested .field.
https://github.com/buefy/buefy/blob/5737a8e1d567899539637ba69af9e24bed32a50e/src/scss/components/_form.scss#L5-L17

To:
```sass
.field {
    &.is-grouped {
        .field {
            flex-shrink: 0;
            margin-right: 0.75rem;
            &.is-expanded {
                flex-grow: 1;
                flex-shrink: 1;
            }
        }
    }
}
```

It is not the best, since there will be margins at the line and wrap ends.

**Comment by yxngl** [comment]
Created a PR. But it will not align on the right when the last field is expanded. For your case, it is probably better to use columns.
